package.json

```diff
@@ -388,262 +395,262 @@
+                    "when": "view == azureResourceGroups && viewItem == applicationSettingItem",
                     "group": "inline"
                 },
                 {
                     "command": "azureFunctions.disconnectRepo",
-                    "when": "view == azFuncTree && viewItem == deploymentsConnected",
+                    "when": "view == azureResourceGroups && viewItem == deploymentsConnected",
                     "group": "1@2"
                 },
                 {
                     "command": "azureFunctions.refresh",
-                    "when": "view == azFuncTree && viewItem =~ /^deployments(C|Unc)onnected$/",
+                    "when": "view == azureResourceGroups && viewItem =~ /^deployments(C|Unc)onnected$/",
                     "group": "2@1"
                 },
                 {
                     "command": "azureFunctions.viewDeploymentLogs",
-                    "when": "view == azFuncTree && viewItem =~ /^deployment//",
+                    "when": "view == azureResourceGroups && viewItem =~ /^deployment//",
                     "group": "1@1"
                 },
                 {
                     "command": "azureFunctions.redeploy",
-                    "when": "view == azFuncTree && viewItem =~ /^deployment//",
+                    "when": "view == azureResourceGroups && viewItem =~ /^deployment//",
                     "group": "1@2"
                 },
                 {
                     "command": "azureFunctions.openInPortal",
-                    "when": "view == azFuncTree && viewItem =~ /^deployment//",
+                    "when": "view == azureResourceGroups && viewItem =~ /^deployment//",
                     "group": "1@3"
                 },
                 {
                     "command": "azureFunctions.viewCommitInGitHub",
-                    "when": "view == azFuncTree && viewItem == deployment/github",
+                    "when": "view == azureResourceGroups && viewItem == deployment/github",
                     "group": "1@4"
                 },
                 {
                     "command": "azureFunctions.refresh",
-                    "when": "view == azFuncTree && viewItem == siteFiles",
+                    "when": "view == azureResourceGroups && viewItem == siteFiles",
                     "group": "1@1"
                 },
                 {
                     "command": "azureFunctions.refresh",
-                    "when": "view == azFuncTree && viewItem == logFiles",
+                    "when": "view == azureResourceGroups && viewItem == logFiles",
                     "group": "1@1"
                 },
                 {
                     "command": "azureFunctions.refresh",
-                    "when": "view == azFuncTree && viewItem == folder",
+                    "when": "view == azureResourceGroups && viewItem == folder",
                     "group": "1@1"
                 }
             ],
```

Comment:
Below (can't add the comment there) the view registration for `azFuncTree` can be removed.